lib/response/CallResponse.js

```diff
@@ -41,8 +35,7 @@ function subscribeToResponse(observer) {
     // Precisely the same error as the router when a call function does not
     // exist.
     if (!model._source) {
-        observer.onError(new Error('function does not exist'));
-        return;
+        return observer.onError(new Error("function does not exist"));
```

Comment:
I think we should leave this as is. It's just more explicit that it returns "undefined" if it's on a separate line.

I was thinking about just adding an eslint comment override, to consciously acknowledge that we're returning 2 different values, and we're ok with it.